pkg/slareport/prom.go

```diff
@@ -18,7 +18,7 @@ var (
 	metricQuery = `(1 - max without(prometheus_replica) (sum_over_time(slo:sli_error:ratio_rate5m{sloth_service=~"appcat-.+"}[{{DURATION}}])
 / ignoring (sloth_window) count_over_time(slo:sli_error:ratio_rate5m{sloth_service=~"appcat-.+"}[{{DURATION}}])
 ) >= 0) * 100`
-	slaQuery         = `100*slo:objective:ratio{sloth_id="{{SLOTHID}}"}`
+	slaQuery         = `sla:objective:ratio{sloth_id="{{SLOTHID}}"}`
```

Comment:
You did not add the `sloth_id` to your recording rule. This metric will only have a `service` label.